### DISCUSSION TITLE
Is the prometheus SeaweedFS_master_is_leader meant to be output on all services?

### DISCUSSION THREAD

**Discussion by adasauce** [seaweedfs/seaweedfs#5314] [Q&A]
I notice that this section is included in the metrics output for master, volume, and filer services that are running.

```
# HELP SeaweedFS_master_is_leader is leader
# TYPE SeaweedFS_master_is_leader gauge
SeaweedFS_master_is_leader 0
```

is the intended behaviour? there are no other `SeaweedFS_master_*` metrics included in the output of the volume or filer services. Initially I thought I was accidentally running masters along with the filer and volume services but I think there would be a lot more havoc in my cluster that's working fine.

**Comment by SmsS4** [comment]
I don't know if it's intended or not but we are using this to find labels of all our seaweed component in kubernetes cluster:)

**Comment by chrislusf** [comment]
It was added in https://github.com/seaweedfs/seaweedfs/pull/2613/files and looked fine. The master/volume/filer server metrics are all present.

cc @kmlebedev
